Gems/PhysX/Code/Source/EditorColliderComponent.cpp

```diff
@@ -466,6 +468,7 @@ namespace PhysX
 
         UpdateShapeConfigurationScale();
         CreateStaticEditorCollider();
+        ValidateRigidBodyMeshGeometryType();
```

Comment:
This process in the function OnConfigurationChanged() will lead to the entire-tree refresh which makes some widget like tag shown in the picture lose focus once I change the text content even if I input just one character. Is there some way to fix it? Look forward to your reply. Thanks.
![image](https://user-images.githubusercontent.com/49116245/143670639-58179a2e-be0c-4110-b96e-0f0caa839220.png)

Reply:
Thanks for bringing this up @MoMoPF. The short answer is no (if for this change we continue to call RefreshEntireTree, as the whole widget is completely destroyed and recreated under the hood).

There are less invasive refresh options, but they aren't always what you need. It might be possible to have `ValidateRigidBodyMeshGeometryType` take a parameter for the refresh type, so in this instance we just call `Refresh_AttributesAndValues` or `Refresh_Values`. It's just we have to be super careful with that as it can then cause other issues if more actually does need to be refreshed.

Please feel free to take a look and experiment with the different options to see, it might be not that hard to do potentially but I'm not sure who is going to be able to look at this in the near-term sorry.

Thanks for raising it though, there are plans afoot to replace a lot of this UX code with a new system in future but it's still quite early on, hopefully the new approach will do a much better job of handling these sorts of things (more idiomatic Qt ideally).